psp-template.md

```diff
@@ -0,0 +1,27 @@
+# Title
+
+* **PSP Number:** [To be assigned]
+* **Authors:** [Name and Email or GitHub Username]
+* **Status:** [Draft, Call for Feedback, Accepted, Integrated]
+* **Created:** [yyyy-mm-dd]
+* **Reference Implementation** [Link to a first reference implementation]
+
+## Summary
```

Comment:
We may want to go through making the ERC-20 equivalent and use it as a reference, then link to it here as a "good example".